app/services/organizations/update_service.rb

```diff
@@ -43,6 +43,10 @@ def call
         )
       end
 
+      if params.key?(:net_payment_term)
+        Organizations::UpdateInvoicePaymentDueDateService.call(organization:, net_payment_term: params[:net_payment_term])
```

Comment:
Layout/LineLength: Line is too long. [122/120] (https://rubystyle.guide#max-line-length)